futures-util/src/compat/tokio.rs

```diff
@@ -14,33 +14,27 @@ use tokio_executor::{DefaultExecutor, Executor as TokioExecutor};
 /// # Examples
 ///
 /// ```
-/// #![feature(async_await, await_macro, futures_api, pin)]
+/// #![feature(async_await, await_macro, futures_api)]
+/// use futures::future::{FutureExt, TryFutureExt};
 /// use futures::spawn;
-/// use futures::channel::oneshot;
 /// use futures::compat::TokioDefaultSpawner;
-/// use futures::executor::block_on;
-/// use futures::future::{FutureExt, TryFutureExt};
-/// use std::thread;
-///
-/// let (sender, receiver) = oneshot::channel::<i32>();
-///
-/// thread::spawn(move || {
```

Comment:
Looks fine (I still think it would be good to have an example of cross-executor communication somewhere, but makes sense to not include it in the basic example here).